pulsar-proxy/pom.xml

```diff
@@ -165,5 +165,20 @@
       <groupId>com.beust</groupId>
       <artifactId>jcommander</artifactId>
     </dependency>
+
+    <dependency>
```

Comment:
Are these new deps needed?

Reply:
removed it they are required for running it in the ide only.